.github/files/jetpack-staging-sites/run-atomic-deploy-e2e-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# This script is ran as part of the `UpdateJetpackStaging` TeamCity build.
+# It triggers an E2E test run against pre-defined atomic sites that have been updated with the newest Jetpack build.
+
+if [ -z "$SIGNATURE_KEY" ] || [ -z "$TRIGGER_URL" ]; then
+  echo "Missing required ENV variables: both 'SIGNATURE_KEY' and 'TRIGGER_URL' are required. Aborting."
+  exit 1
+fi
+
+echo "Starting atomic deploy E2E tests..."
+
+REQUEST_SIGNATURE=$(echo -n | openssl dgst -sha256 -hmac "$SIGNATURE_KEY")
+curl --fail-with-body -s -X POST -H "X-Jetpack-Atomic-Deploy-E2E-Signature: $REQUEST_SIGNATURE" "$TRIGGER_URL"
```

Comment:
Looks like the command you're using here produces a value for `REQUEST_SIGNATURE` along the lines of "SHA2-256(stdin)= b613679a0814d9ec772f95d778c35fc5ff1697c493715653c6c712144292c5ad", which of course doesn't work due to that "SHA2-256(stdin)=" prefix.

Something like this should work
```suggestion
REQUEST_SIGNATURE=$(echo -n | openssl mac -digest sha256 -macopt "key:$SIGNATURE_KEY" HMAC)
curl --fail-with-body -s -X POST -H "X-Jetpack-Atomic-Deploy-E2E-Signature: ${REQUEST_SIGNATURE,,}" "$TRIGGER_URL"
```

Reply:
Ahhh, thank you so much for catching that! 🙂  That's really weird, because I ran this in Bash on my local system and wasn't seeing the prefix attached in that value. 🤔  But that would for sure explain why that wasn't working here!

Reply:
Different version of openssl maybe? I have 3.0.10, and the Actions runner probably has [3.0.2](https://packages.ubuntu.com/jammy/openssl).

Reply:
Ahh yeah, that's probably it, good point!